plugin/trino-delta-lake/src/test/java/io/trino/plugin/deltalake/TestDeltaLakeConnectorTest.java

```diff
@@ -2208,7 +2208,7 @@ public void testProjectionPushdownNonPrimitiveTypeExplain()
         assertExplain(
                 "EXPLAIN SELECT id, _row.child, _array[1].child, _map[1] FROM " + tableName,
                 "ScanProject\\[table = (.*)]",
-                "expr(.*) := system\\.builtin\\.\\$operator\\$subscript\\(.*, bigint '1'\\).0",
+                "expr(.*) := .*\\$subscript\\(.*, bigint '1'\\).0",
```

Comment:
Do we really want tests like this?

Reply:
No, I'm thinking about removing it. It's super brittle -- it breaks on every other PR that touches the IR.